pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -175,10 +175,17 @@ func (w *Watcher) init() error {
 }
 
 // Walks through the plugin directory discover any existing plugin sockets.
+// Goroutines started here will be waited for in Stop() before cleaning up.
+// Ignore all errors expect root dir not being walkable
```

Comment:
one minor thing, expect -> except root dir not walkable

Reply:
Fixed